index.html

```diff
@@ -0,0 +1,189 @@
+                    </div>
+                    <div class="commentTextBar">
+                        <i class="fa fa-circle commentIcon"></i>
+                        <input type="text" placeholder="Add a public comment...">
+                    </div>
+
+
+                    <div class="commentWhole">
+                        <i class="fa fa-circle commentIcon"></i>
+                        <div class="commentContainer">
+                            <h4>Username Here <span>2 years ago</span></h4>
+                            <span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus in neque sit amet massa ullamcorper finibus. Vivamus commodo leo ut ante venenatis rhoncus. Quisque non dolor eget felis tincidunt tempus eget a mi. </span>
+                            <div class="commentLikeReply">
+                                <i class="fa fa-thumbs-up"> 3</i>
+                                <i class="fa fa-thumbs-down"></i>
+                                <span>REPLY</span>
+                            </div>
+                        </div>
+                    </div>
+
+
+                    <div class="commentWhole">
+                        <i class="fa fa-circle commentIcon"></i>
+                        <div class="commentContainer">
+                            <h4>Username Here <span>2 years ago</span></h4>
+                            <span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus in neque sit amet massa ullamcorper finibus. Vivamus commodo leo ut ante venenatis rhoncus. Quisque non dolor eget felis tincidunt tempus eget a mi. </span>
+                            <div class="commentLikeReply">
+                                <i class="fa fa-thumbs-up"> 3</i>
+                                <i class="fa fa-thumbs-down"></i>
+                                <span>REPLY</span>
+                            </div>
+                        </div>
+                    </div>
+
+
+                    <div class="commentWhole">
+                            <i class="fa fa-circle commentIcon"></i>
+                            <div class="commentContainer">
+                                <h4>Username Here <span>2 years ago</span></h4>
+                                <span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus in neque sit amet massa ullamcorper finibus. Vivamus commodo leo ut ante venenatis rhoncus. Quisque non dolor eget felis tincidunt tempus eget a mi. </span>
+                                <div class="commentLikeReply">
+                                    <i class="fa fa-thumbs-up"> 3</i>
+                                    <i class="fa fa-thumbs-down"></i>
+                                    <span>REPLY</span>
+                                </div>
+                            </div>
+                        </div>
+                </article>
+            </section>
+            <aside> <!-- Up Next Sidebar thing-->
```

Comment:
Using Aside is a bit old school, we could use section instead, but no biggie there, you could leave it like that